clang/include/clang/Sema/Sema.h

```diff
@@ -1200,6 +1200,27 @@ class Sema final {
   //
   //
 
+  /// \name Sema Components
+  /// Parts of Sema
+  ///@{
+
+  // Just in this section, private members are followed by public, because
+  // C++ requires us to create (private) objects before (public) references.
+
+private:
+  std::unique_ptr<SemaOpenACC> OpenACCPtr;
```

Comment:
A comment describing how this works would be nice for the folks that come around here in the future.

Reply:
Now with reference replaced by function, do you still think comment is needed?

Reply:
> Now with reference replaced by function, do you still think comment is needed?

Nope!  Much less problematic now.